hardware/arduino/avr/boards.txt

```diff
@@ -28,7 +28,7 @@ yun.upload.wait_for_upload_port=true
 yun.bootloader.tool=avrdude
 yun.bootloader.low_fuses=0xff
 yun.bootloader.high_fuses=0xd8
-yun.bootloader.extended_fuses=0xfb
+yun.bootloader.extended_fuses=0xcb
```

Comment:
Is this correct? I don't really understand why, but this was changed from 0xcb to 0xfb in https://github.com/arduino/Arduino/commit/e745ed988fd5f011e444a775ba31b6bb0c3dc4ed. It would have been nice if that commit message actually gave some information on the reason for the change. This forum thread gives some info: http://forum.arduino.cc/index.php?topic=223157.msg1708120#msg1708120. This issue also looks relevant: https://github.com/arduino/openwrt-yun/issues/12.

0xfb doesn't verify with an external programmer but it didn't with avrdude 6.0.1-arduino5 either.